src/app/pages/admin-panel/admin-panel.component.ts

```diff
@@ -1,31 +1,12 @@
 import { Component, OnInit } from '@angular/core';
-import { AuthService } from '../../core/service/auth.service';
-import { Router } from '@angular/router';
-import { ToastService } from 'src/app/core/service/toast.service';
 
 @Component({
   selector: 'app-admin-panel',
   templateUrl: './admin-panel.component.html',
   styleUrls: ['./admin-panel.component.css'],
 })
 export class AdminPanelComponent implements OnInit {
-  constructor(
-    private authService: AuthService,
-    private router: Router,
-    private toastService: ToastService
-  ) {}
+  constructor() {}
```

Comment:
if it is unused. It is not necessary to keep this in the code

Reply:
@aky-wtag vai resolved it